src/state/stableswap/constants.ts

```diff
@@ -208,6 +212,29 @@ export const STABLESWAP_POOLS: StableSwapPools = {
     route: 'usd',
     isOutdated: false,
     rewardPids: null
+  },
+  [StableSwapPoolName.AXLUSD_USDC_USDT_USN]: {
+    name: StableSwapPoolName.AXLUSD_USDC_USDT_USN,
+    friendlyName: 'axlUSDC-USDC/USDT/USN',
+    lpToken: new Token(
+      ChainId.AURORA,
+      '0x62A0C1Bb3ff819d7ad8B1d69C2270f473aeF6f55',
+      18,
+      'axlUSDC-USDC/USDT/USN TLP',
+      'Trisolaris axlUSDC-USDC/USDT/USN'
+    ),
```

Comment:
Maybe move this to `dev/generate_tokens/base_tokens_map.js`  so it's variablized and the token name is consistent across the codebase